### PR TITLE
redid liver analysis with new pseudobulks and added bayesprism analysis

### PR DESCRIPTION
**Pull request by nrosed**
In this PR I updated the data used for the liver analysis and included a new comparator, BayesPrism. 

I had to redo all downstream liver analyses because of the new liver data, but no big changes in performance. So there isn't a lot of code changed in the notebooks, only re-running and generating new figures.

### CODE REVIEW

File: bayesprism_scripts/run_kang_bayes_prism.sh
Comment (ivichadriana) [inline bayesprism_scripts/run_kang_bayes_prism.sh:1]: you have commented## lines here, just a heads up

File: bayesprism_scripts/run_liver_bayes_prism.sh
Comment (ivichadriana) [inline bayesprism_scripts/run_liver_bayes_prism.sh:1]: comments here too, up to you if you want to keep them

**Review by ivichadriana — APPROVED** [review]
two shell scripts have some commented lines that I pointed out; other than that everything looks good with the new data.

File: bayesprism_scripts/run_kang_bayes_prism.sh
Comment (nrosed) [reply bayesprism_scripts/run_kang_bayes_prism.sh:1]: deleted the commented out lines

File: bayesprism_scripts/run_liver_bayes_prism.sh
Comment (nrosed) [reply bayesprism_scripts/run_liver_bayes_prism.sh:1]: deleted the commented out lines

**Pull request merged**